api/src/main/java/org/apache/iceberg/encryption/EncryptedOutputFile.java

```diff
@@ -37,4 +37,9 @@ public interface EncryptedOutputFile {
    * #encryptingOutputFile()}.
    */
   EncryptionKeyMetadata keyMetadata();
+
+  /** Underlying output file for native encryption. */
+  default OutputFile rawOutputFile() {
```

Comment:
I'm not sure about this name, but I've had a hard time coming up with a better one. The best I can come up with is `plainOutputFile`. What do you think about that? I think it's slightly clearer than "raw".